### DISCUSSION TITLE
shottr

### DISCUSSION THREAD

**Discussion by Schnauzer-World** [Homebrew/discussions#4680] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.2
ORIGIN: https://github.com/Homebrew/brew
HEAD: 3db1acf3e38e270af4e1c3f214622bbfb18f830e
Last commit: 4 days ago
Core tap JSON: 31 Jul 00:55 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 16
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 16-core 64-bit cometlake
Clang: 14.0.0 build 1400
Git: 2.37.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 12.6.8-x86_64
CLT: 14.2.0.0.1.1668646533
Xcode: N/A
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

SHA256 mismatch
Monterey 12.6.8

### Relevant casks

shottr

**Comment by osalbahr** [comment]
[Cask - checksum does not match](https://docs.brew.sh/Common-Issues#cask---checksum-does-not-match)

Let me know if this helps.

**Reply by Schnauzer-World** [reply]
Thanks. I would love to learn more and contribute.

I ran 'brew fetch --force shottr'

/Users/username/Library/Caches/Homebrew/downloads/6e1b98198506e18090539fc7811f0c2af810ba23413ad53e17170c713bd9c451--Shottr-1.7.1.dmg
SHA256: 99ee3c3683185e690e665cbdda89a4059e89ec6bf2e21415f9adfe3bb43606bf
Warning: Cask reports different sha256: bb6e31ba80a3bd0dfee5ca9b3d98a3409abe9c450cec6ee37b2f558b344a19b1


Looks like checksum needs updated.

**Reply by osalbahr** [reply]
You can follow [How to Open a Pull Request (and get it merged)](https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request) to guide you through the process.
